pyo3-ffi/src/cpython/pyhash.rs

```diff
@@ -0,0 +1,30 @@
+#[cfg(Py_3_13)]
+use crate::PyObject;
+use crate::{Py_hash_t, Py_ssize_t};
+use std::os::raw::{c_char, c_int, c_void};
+
+#[repr(C)]
+#[derive(Copy, Clone)]
```

Comment:
```suggestion
#[derive(Copy, Clone)]
#[cfg(not(any(PyPy, GraalPy)))]`
```

... and similar for `PyHash_GetFuncDef`, perhaps.